src/scenes/varrock-anvil.js

```diff
@@ -0,0 +1,25 @@
+import { CONSTANTS } from "../constants/constants.js";
+import { LevelScene } from "./level.js";
+import { Anvil } from "../targets/anvil.js";
+
+export class VarrockAnvilScene extends LevelScene {
+    constructor() {
+        super({
+            levelType: CONSTANTS.LEVEL_TYPE.SMITHING,
```

Comment:
Need to add this to the check in `AutoClicker::start()`.  to prevent getting an error when autoclickers/clan members try to "click" on the target.
![image](https://user-images.githubusercontent.com/9402065/116798547-2d8e4380-aab6-11eb-8905-25eeeb5acef4.png)

![image](https://user-images.githubusercontent.com/9402065/116798498-8d381f00-aab5-11eb-8ee6-e4f8c132498b.png)

Reply:
I was able to fix this I think? I was able to stop the error with the bot. I'm not sure 100% how auto-clickers work yet